python-threatexchange/threatexchange/cli/match_cmd.py

```diff
@@ -196,33 +214,86 @@ def execute(self, settings: CLISettings) -> None:
             for s_type, index in indices:
                 seen = set()  # TODO - maybe take the highest certainty?
                 if self.as_hashes:
-                    results = _match_hashes(path, s_type, index)
+                    results_from_hashes = _match_hashes(path, s_type, index)
+                    results: t.Sequence[IndexMatchWithRotation] = [
+                        IndexMatchWithRotation(match=match)
+                        for match in results_from_hashes
+                    ]
```

Comment:
I created a new dataclass IndexMatchWithRotation which will be the return type of _match_file.

Q: Should I also make it the return type of _match_hashes? Just so the return types of these 2 are similar

Reply:
To make the python typing pass you may need to be consistent between them (see what mypy says)